main.tf

```diff
@@ -395,3 +401,11 @@ resource "aws_route53_record" "master" {
   records = ["${aws_eip.master.public_ip}"]
   ttl     = 300
 }
+
+resource "aws_route53_record" "master-internal" {
+  zone_id = "${data.aws_route53_zone.dns_zone.zone_id}"
+  name    = "internal.${var.cluster_name}.${var.hosted_zone}"
+  type    = "A"
+  records = ["${aws_network_interface.master_private_interfaces.*.private_ips}"]
```

Comment:
Why do we need the network interface here? Why is not sufficient to just use `${aws_instance.master.private_ip}` for the internal host name? AFAIK They should not need additional Network Interface to see each other.